Sources/SwiftInspectorCommands/Tests/TypealiasCommandSpec.swift

```diff
@@ -50,24 +50,44 @@ final class TypealiasCommandSpec: QuickSpec {
       }
 
       context("with no --name argument") {
-        it("fails") {
-          let result = try? TestTask.run(withArguments: ["typealias", "--path", path])
-          expect(result?.didFail) == true
+        var result: TaskStatus?
+        beforeEach {
+          result = try? TestTask.run(withArguments: ["typealias", "--path", path])
+        }
+
+        it("succeeds") {
+          expect(result?.didSucceed) == true
+        }
+
+        it("outputs the typealias information") {
+          expect(result?.outputMessage).to(contain("SomeAlias"))
+        }
+
+        it("does not output the typelias identifiers") {
+          expect(result?.outputMessage).toNot(contain("SomeType"))
         }
       }
 
       context("with an empty --name argument") {
-        it("fails") {
+        it("succeeds") {
           let result = try? TestTask.run(withArguments: ["typealias", "--name", "", "--path", path])
+          expect(result?.didSucceed) == true
+        }
+      }
+
+      context("with an empty --path argument") {
+        it("fails") {
+          let result = try? TestTask.run(withArguments: ["typealias", "--path", ""])
           expect(result?.didFail) == true
         }
       }
 
       context("with all arguments") {
```

Comment:
I think it would be nice to be more specific to future proof this. If you add a new argument you may forget to update this. Just a thought.

Reply:
https://github.com/fdiaz/SwiftInspector/pull/10/commits/dc51dcba4124c745d567d7785379f6e5f868cf78